### PR TITLE
update readme.md with developer instructions for setup front-end repo

### PR DESCRIPTION
**Pull request by mariuscmoldovan**
## Description
Update readme.md with developer instructions for setup the front-end repo and creating a .env file. 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x ] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,10 +29,12 @@ You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste
 
 🚫 more info on using badges [here](https://github.com/badges/shields)
 
-- Fork and clone the repo to install it as your own remote.
-  - **note** please [be sure to set your remote](https://help.github.jp/enterprise/2.11/user/articles/changing-a-remote-s-url/) for this repo to point to your Labs Team Front End Repository.
-- run: `npm install` to download all dependencies.
-- run: `npm start` to start your local development server.
+## Developer Instructions
+
+1. Clone the repo [front-end](https://github.com/Lambda-School-Labs/underdog-devs-fe-a) repositorie to your machine. DO NOT FORK.
```
Comment (kennymiesner) [inline README.md:34]: Update 'repositorie' to 'repository'

**Review by kennymiesner — CHANGES_REQUESTED** [review]
The only thing I noticed was a spelling error on line 34. Added a comment for more clarification. Thank you for taking some time to get the README updated!

**Review by iwood55 — APPROVED** [review]
Necessary and insightful changes done to the README, well done.

**Review by mariuscmoldovan — COMMENTED** [review]
done

File: README.md (hunk 1 above)
Comment (tfbielawski) [reply README.md]: Thanks!

**Review by kennymiesner — APPROVED** [review]

**Pull request merged**